src/docs/asciidoc/setting_up_clusters.adoc

```diff
@@ -336,26 +336,26 @@ so that you can isolate your clusters.
 * `port`: Integer value that is used to set the multicast port.
 
 
-[[creating-cluster-groups]]
-=== Creating Cluster Groups
+[[creating-cluster]]
+=== Creating Cluster
 
```

Comment:
Let's keep the [creating-cluster-groups] anchor as it is, not to get 404s for the existing links to this section.